src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string | null,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    key: null,
+  };
+
+  message = 'Nothing was pressed yet';
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ key: event.key });
+    });
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/214508306-ea108bac-c3c0-4bc1-81de-816066675879.png)
Also, I think it's better to save the function in some method